ansible/roles/debops.apt_mark/files/script/apt-mark-status3

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+from __future__ import print_function
+from json import dumps
+from subprocess import Popen, PIPE
+import os
+import signal
+
+
+def run_command(command_string):
+    """Gather information about APT packages"""
+    process = Popen([command_string], stdout=PIPE, stderr=PIPE,
+                    shell=True, preexec_fn=os.setsid)
```

Comment:
Please use this opportunity to remove the `shell=True` and pass a list of arguments.

Reply:
You mean [this](https://medium.com/python-pandemonium/a-trap-of-shell-true-in-the-subprocess-module-6db7fc66cdfd)? I'll check it out, thanks.

Reply:
I was more referring to the fix I did in `etckeeper.fact.j2` but that article was interesting as well.

Reply:
@ypid Check if the update works when you have some time.